### ISSUE TITLE
Error reporting on failures isn't helpful

### ISSUE THREAD

**Issue by n1k0** [mozilla/readability#147]
Refs #146. We need to improve error reporting when Readability.js fails:

![screen shot 2015-04-16 at 12 10 49](https://cloud.githubusercontent.com/assets/41547/7178774/a5f1bf98-e431-11e4-9ae4-045b49cc7d72.png)

**Comment by n1k0** [comment]
Patch:

```diff
diff --git a/test/test-readability.js b/test/test-readability.js
index 37b8409..c92d475 100644
--- a/test/test-readability.js
+++ b/test/test-readability.js
@@ -10,6 +10,12 @@ var JSDOMParser = readability.JSDOMParser;
 
 var testPages = require("./utils").getTestPages();
 
+function reformatError(err) {
+  var formattedError = new Error(err.message);
+  formattedError.stack = err.stack;
+  return formattedError;
+}
+
 function runTestsWithItems(label, beforeFn, expectedContent, expectedMetadata) {
   describe(label, function() {
     this.timeout(5000);
@@ -17,7 +23,11 @@ function runTestsWithItems(label, beforeFn, expectedContent, expectedMetadata) {
     var result;
 
     before(function() {
-      result = beforeFn();
+      try {
+        result = beforeFn();
+      } catch(err) {
+        throw reformatError(err);
+      }
     });
 
     it("should return a result object", function() {
```

Will send a PR soon.

**Comment by n1k0** [comment]
Fixed by 4721837e27bb4fdb2bd6676768f14dcc12aed6b8.
